shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsController.h

```diff
@@ -130,7 +129,7 @@
 // If the `PlatformViewsController` does not contain any `FlutterPlatformView` object or
 // a `FlutterPlatformView` object associated with the view_id cannot be found, the method
 // returns nil.
-- (UIView*)platformViewForId:(int64_t)viewId;
+- (UIView* _Nullable)platformViewForId:(int64_t)viewId;
```

Comment:
in the future we may wanna rename this to `viewID`?

Reply:
Probably yeah. I used this to be consistent with the rest of the Obj-C codebase.